docs/site/src/components/apiMenuData.js

```diff
@@ -0,0 +1,68 @@
+// @flow weak
+
+// API menus for components whose entries don't follow `component`, `componentSub1`, `componentSub2` pattern.
+export const apiMenus = {
```

Comment:
What about making everything explicit under a single object.
I'm having some trouble to understand how it's working. 
I think that it can be simpler.

```js
mappingRoutesPropTypes: {
  'layout/responsive-ui': [
    'layout',
  ],
  'component-demos/bottom-navigation': [
    'bottom-navigation',
    'bottom-navigation-button',
  ],
  // ...
}

```